bin/travis-deploy.sh

```diff
@@ -1,31 +1,33 @@
-#! /bin/sh
+#!/usr/bin/env bash
 
+# Show script in output, and error if anything fails
 set -xe
 
-# Update this whenever the latest Node.js LTS version changes (~ every year).
-# Do not forget to add this version to .travis.yml config also.
-LATEST_LTS_VERSION="10"
-
-# We want this command to succeed whether or not the Node.js version is the
-# latest (so that the build does not show as failed), but _only_ the latest
-# should be used to publish the module.
-if [ "$TRAVIS_NODE_VERSION" != "$LATEST_LTS_VERSION" ]; then
```

Comment:
We still need it!

Reply:
@shumkov Not anymore, now it's just Node v10. And it is a service not a library, so I think it should be as in Drive.

Reply:
Yes, but if we want to support all LTS versions of NodeJS?

Reply:
You're right mate, I'll fix it. Sorry for being that sure of myself.

Reply:
No-no man, nothing to say sorry for.